docs/using/syntax.md

```diff
@@ -179,7 +179,7 @@ sections below:
   - See {ref}`syntax/roles` for more
   information.
   - ```md
-    `{rolename}`interpreted text`
```

Comment:
I'm just getting going with MyST-Parser (i.e. far from an expert) but I based this change on https://myst-parser.readthedocs.io/en/latest/using/syntax.html#roles-an-in-line-extension-point (and on empirical usage of the feature).

Thanks a lot for this project!